parsing/parser.mly

```diff
@@ -2482,7 +2481,7 @@ label_declaration_semi:
 
 /* Type Extensions */
 
-str_type_extension:
+%inline str_type_extension:
```

Comment:
These `str_type_extension` and `sig_type_extension` are part of the respective `structure_item` and `signature_item` branchout, I wouldn't inline them.